crates/notion-core/src/session.rs

```diff
@@ -174,29 +175,23 @@ impl Session {
     /// Fetch and unpack a version of Node matching the input requirements.
     pub fn install_node(&mut self, version_spec: &VersionSpec) -> Fallible<()> {
         let node_distro = self.fetch_node(version_spec)?.into_version();
-        let success_message = format!(
-            "Installation successful. Default node version set to {}",
-            &node_distro.runtime
-        );
+        let success_message = format!("installed and set node@{} as default", &node_distro.runtime);
```

Comment:
We can use one of the other `style` helper functions to make sure the tool names here get formatted consistently:

```rust
use crate::style::{format_success_message, tool_version};
```

```rust
let success_message =
    format!("installed and set {} as default", tool_version("node", &node_distro.runtime));
```

Reply:
Good catch, updated!